.github/workflows/CI.yml

```
@@ -68,9 +68,21 @@ jobs:
             --exclude ./**/target ./**/weights \
             -- ./**/src/**/*.rs \
             2>out.txt \
-            && exit 1 || exit 0
+            && exit 1 || true
             # We expected it to fail because there are some unlicensed files left.
           
           grep -q 'No license detected in reconstruct.rs. Exact file path:' ./out.txt
           grep -q 'No license detected in mod.rs. Exact file path:' ./out.txt
         working-directory: polkadot
+      - name: Enforce product references in headers
+        run: |
+          license-scanner scan \
+            --ensure-product 'Polkadot' \
+            -- ./xcm/src/lib.rs \
+            2>out.txt \
+            && exit 1 || true
```

```suggestion
          set -e      
          ! license-scanner scan \
            --ensure-product 'Polkadot' \
            -- ./xcm/src/lib.rs \
            2>out.txt
```
Instead of silencing the failure, this should assert it

---

Right, I missed the simplest way 😅 It didn't need the `set -e`too.